src/components/ha-dialog.ts

```diff
@@ -71,6 +71,10 @@ export class HaDialog extends Dialog {
           position: var(--dialog-surface-position, relative);
           top: var(--dialog-surface-top);
           min-height: var(--mdc-dialog-min-height, auto);
+          border-radius: var(
+            --ha-card-border-radius,
+            var(--mdc-shape-medium, 4px)
+          );
```

Comment:
```suggestion
          border-radius: var(
            --ha-dialog-border-radius,
            var(--ha-card-border-radius, 4px)
          );
```

Reply:
Ah, then I misunderstood what you wrote above. I thought you were against a `ha-dialog` var in general and wanted to use the existing `ha-card` one instead to not have too many vars 😆.